docs/pages/introduction/expo.md

```diff
@@ -0,0 +1,44 @@
+---
+title: What is Expo
+---
+
+import { InlineCode } from '~/components/base/code';
+import { BoxLink } from '~/ui/components/BoxLink';
+import { YesIcon, NoIcon } from '~/ui/components/DocIcons';
+
+Expo is an npm package that enables a suite of incredible features for React Native apps. The `expo` package can be installed in nearly any React Native project.
+
+## Features
+
+<BoxLink title="Expo Go" description="Share your development app instantly with your team from anywhere in the world." href="workflow/expo-go" />
+<BoxLink title="Expo SDK" description="Comprehensive suite of well tested React Native modules that run on iOS, Android, and web." href="versions/latest" />
+<BoxLink title="Native API" description="Write highly performant native code with modern Swift & Kotlin API." href="modules/overview" />
+<BoxLink title="Prebuild" description="Separate React from Native to develop from any computer, upgrade easily, white label apps, and maintain larger projects." href="workflow/prebuild" />
+<BoxLink title="Expo CLI" description="Manage dependencies, compile native apps, develop for web, and connect to any device with a powerful dev server." href="workflow/expo-cli" />
```

Comment:
```suggestion
<BoxLink title="Expo Go" description="Share your development app instantly with your team from anywhere in the world." href="/workflow/expo-go" />
<BoxLink title="Expo SDK" description="Comprehensive suite of well tested React Native modules that run on iOS, Android, and web." href="/versions/latest" />
<BoxLink title="Native API" description="Write highly performant native code with modern Swift & Kotlin API." href="/modules/overview" />
<BoxLink title="Prebuild" description="Separate React from Native to develop from any computer, upgrade easily, white label apps, and maintain larger projects." href="/workflow/prebuild" />
<BoxLink title="Expo CLI" description="Manage dependencies, compile native apps, develop for web, and connect to any device with a powerful dev server." href="/workflow/expo-cli" />
```

The links do not work since they all use `/introduction/` as the path.